src/main/java/com/lilybookclub/entity/BaseEntity.java

```diff
@@ -0,0 +1,37 @@
+package com.lilybookclub.entity;
+
+import com.lilybookclub.enums.Status;
+import jakarta.persistence.*;
+import lombok.Data;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.LastModifiedBy;
+
+import java.time.LocalDateTime;
+
+@MappedSuperclass
+@Data
+public abstract class BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private Status status;
```

Comment:
either you have status or deleted. They seem to fulfil similar purpose. Lets stick with status

Also pls read up on `@Enumerated` and kets use the string version